spec/controllers/api/allocation_api_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Api::AllocationApiController, type: :controller do
```

Comment:
There is also a spec file testing existing API functionality located here:

```
spec/api/allocation_api_spec.rb
```

The format of the spec is slightly different from what I'm used to – I think it's also used to generate swagger documentation. It might be worth checking with @starswan to see if this test should actually be consolidated into that one...